solvers/geomloss.py

```diff
@@ -32,11 +32,16 @@ class Solver(BaseSolver):
     # the cross product for each key in the dictionary.
     parameters = {
         'reg': [1e-2, 1e-1],
-        'use_gpu': [True],
+        'use_gpu': [True, False],
     }
 
     stopping_criterion = SufficientProgressCriterion(patience=50)
 
+    def skip(self, **kwargs):
+        # we skip the solver if use_gpu is True and no GPU is available
+        if self.use_gpu and not torch.cuda.is_available():
+            return True, "No GPU available"
```

Comment:
This will fail if a gpu is available:
```suggestion
            return True, "No GPU available"
        return False, None
```

Yes I know, the API is not great, it would be better to simply return the reason if skipping, else  `None`.